beacon-chain/rpc/eth/validator/validator.go

```diff
@@ -586,6 +634,21 @@ func (vs *Server) ProduceBlindedBlockSSZ(ctx context.Context, req *ethpbv1.Produ
 			Data:    sszBlock,
 		}, nil
 	}
+	capellaBlock, ok := v1alpha1resp.Block.(*ethpbalpha.GenericBeaconBlock_Capella)
+	if ok {
+		block, err := migration.V1Alpha1BeaconBlockCapellaToV2(capellaBlock.Capella)
```

Comment:
I am not sure this is correct, I could not find the Capella equivalent of `V1Alpha1BeaconBlockBellatrixToV2Blinded`

Reply:
You need to write this function. Fortunately I already have it here: https://gist.github.com/rkapka/222a4224be3ccf64808528927e74d881